src/Files.App/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -382,26 +400,34 @@
 					<ListView.ItemTemplate>
 						<DataTemplate x:DataType="ICollectionViewGroup">
 							<StackPanel
-								Margin="0,4,0,4"
+								Margin="0,4"
```

Comment:
Can you revert? Coding convention is to explicitly define the margin. The same goes for grid rows (we always define `0` even though it's the default.